### ISSUE TITLE
Maintaining this repository

### ISSUE THREAD

**Issue by faddat** [line/lbm-sdk#533]
Hi!

Seems you've done quite a bit of work to get to this point!

I don't know if you have also forked tendermint but I would like to make some recommendations about maintaining this code. In my experience maintaining forks of the cosmos SDK, things get tough pretty fast. If I understand correctly, your chain has not yet launched. That is good. I would like to recommend that you try to launch on cosmos SDK 46 derived code.

Over time, the number one issue that I encounter as an infrastructure operator and validator is performance. STK 46 is likely to improve performance a good deal. I am running a public test net on it now.

Typically, the best technique is to merge the foreign code base into this one frequently, so that you can pull in upgrades. It is very important to note that very frequently those upgrades are security related in nature, and that he will want to apply them rapidly. As for whether to build your modules in this repository or in the repository of chains that will be built with this code, that is really a difficult question. Generally speaking my personal practice is to put new functionality inside the chain itself, and make changes in the SDK.

If you are planning to use IBC, you want to keep up with the IBC – go repository as well.

So to summarize:

* recommend launching on sdk 46 / tm 45 / ibc-go >v3
* recommend that you make it easy for yourselves to merge in updates from the cosmos-sdk
* recommend that you do as little original code work in this repository as possible, saving features for chains, and later if desiring to reuse modules, importing them.

**Comment by zemyblue** [comment]
Thank you for your kind comments when operating the repository. 

As you said, there are many things to consider in many ways to fork and operate comsos-sdk. And if there is a chain that operates even with the latest cosmos-sdk as it is, there are parts that need to be more careful.
We developed a fork at 0.38, and we are monirtoring the progress of 0.46 as there are many changes. And we'll check the change of 0.46 more aggressively.

Thank you.
